linode/nbtypes/framework_models.go

```diff
@@ -0,0 +1,121 @@
+package nbtypes
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+	"github.com/linode/terraform-provider-linode/v2/linode/helper/frameworkfilter"
+)
+
+type DataSourceModel struct {
+	ID           types.String `tfsdk:"id"`
+	Label        types.String `tfsdk:"label"`
+	Price        types.List   `tfsdk:"price"`
+	RegionPrices types.List   `tfsdk:"region_prices"`
+	Transfer     types.Int64  `tfsdk:"transfer"`
+}
+
+func (data *DataSourceModel) ParseNodeBalancerType(nbType *linodego.NodeBalancerType,
```

Comment:
Should be the same not to expose the functions in this `framework_models` and the rest of the data sources.

Reply:
Fixed!